pkgs/development/python-modules/flask-appbuilder/default.nix

```
@@ -20,22 +20,21 @@
 , python-dateutil
 , prison
 , pyjwt
+, pyyaml
 , sqlalchemy-utils
 }:
 
 buildPythonPackage rec {
   pname = "flask-appbuilder";
-  version = "3.1.1";
+  version = "3.2.3";
 
   src = fetchPypi {
     pname = "Flask-AppBuilder";
     inherit version;
-    sha256 = "076b020b0ba125339a2e710e74eab52648cde2b18599f7cb0fa1eada9bbb648c";
+    sha256 = "sha256-+ZYrn2LnVORyYsnZtsH3JX+4XbGgAZZ/Eh6O5gUP+y4=";
   };
 
-  checkInputs = [
-    nose
-  ];
+  patches = [ ./flask-jwt-extended-4.1-compat.patch ];
```

With something like the example below we don't need to add the patch to our repo.

```suggestion
  patches = [
    # Upgrade flask_jwt_extended and PyJWT to the latest versions, https://github.com/dpgaspar/Flask-AppBuilder/pull/1610
    (fetchpatch {
      name = " flask_jwt_extended-and-pyjwt.patch";
      url = "https://github.com/dpgaspar/Flask-AppBuilder/commit/7097a7b133f27c78d2b54d2a46e4a4c24478a066.patch";
      sha256 = "000000000000000000000000000000000000000000000000000";
    })
  ];
```